src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -314,7 +314,6 @@
 
 {%- block widget_attributes -%}
     id="{{ id }}" name="{{ full_name }}"
-    {%- if read_only %} readonly="readonly"{% endif -%}
     {%- if disabled %} disabled="disabled"{% endif -%}
```

Comment:
I'm sorry I don't think I'll have time to test, but I believe the test that was changed in this pull request could be used to check? (i.e. revert the test and see if it fails/passes)